tox.ini

```diff
@@ -1,24 +1,20 @@
 [tox]
-envlist = py{27,35}-django{111}, py{35,36}-django{20,21,22},quality,docs
+envlist = , py{35,38}-django{22,30},quality,docs
```

Comment:
Django 3.0 with Python3.5 is an invalid combination, also I think that there is an additional ,